action.yml

```diff
@@ -0,0 +1,134 @@
+name: 'Build, test, package'
+description: 'Build, test, and package distributions for an ITK external module.'
+inputs:
+  c-compiler:
+    required: true
+  cxx-compiler:
+    required: true
```

Comment:
Can we make these optional? In a step, if they are an empty string, set to the per-platform default.

Reply:
@thewtex As far as I can tell the only way to set platform-dependent variables without a matrix is by [updating environment variables](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable) in separate steps conditioned on the runner OS. It could quickly become burdensome to have four steps for each parameter, one to check whether a value was passed and three to set platform-specific values if no value was passed in.

For the sake of brevity I've updated `action.yml` so that `env.C-COMPILER` and `env.CXX-COMPILER` are always set based on the OS and are not accepted as input parameters. 

Tests are passing at [https://github.com/InsightSoftwareConsortium/ITKSplitComponents/runs/7953813843?check_suite_focus=true](https://github.com/InsightSoftwareConsortium/ITKSplitComponents/runs/7953813843?check_suite_focus=true). Strangely, Linux CI is cancelling automatically but seems to be unrelated to these changes as I was able to reproduce by rerunning CI in `master` ([https://github.com/InsightSoftwareConsortium/ITKSplitComponents/runs/7953866746?check_suite_focus=true](https://github.com/InsightSoftwareConsortium/ITKSplitComponents/runs/7953866746?check_suite_focus=true)).

Reply:
@thewtex Does this satisfy your requested change?